key-export-client/src/lib.rs

```diff
@@ -84,20 +81,20 @@ impl KeyExportContext {
     /// export the key of the wallet with the given `wallet_id`.
     ///
     /// Throws `Error` in case of failure.
-    pub fn build_key_export_request(&self) -> Result<utils::RequestType, utils::ErrorType> {
+    pub fn build_key_export_request(&self) -> Result<types::RequestType, types::ErrorType> {
```

Comment:
Since the module is called `types`, and you prefix it everywhere, maybe it makes sense to remove `Type` suffix? I.e.
```suggestion
    pub fn build_key_export_request(&self) -> Result<types::Request, types::Error> {
```